### ISSUE TITLE
Burn Docs Icon Color Contrast

### ISSUE THREAD

**Issue by softmaximalist** [tracel-ai/burn#4249]
In the https://burn.dev/docs/burn/ website, the color of the navigation icons at the top right is too similar to the background color. This results in low contrast making it less accessible. To keep coloring consistent with the other icons in the doc, change the color from black to white. 

<img width="3840" height="1854" alt="Image" src="https://github.com/user-attachments/assets/b2f2a7fd-002d-4e28-9e81-09288bb850bf" />

**Comment by arferreira** [comment]
> In the https://burn.dev/docs/burn/ website, the color of the navigation icons at the top right is too similar to the background color. This results in low contrast making it less accessible. To keep coloring consistent with the other icons in the doc, change the color from black to white.
> <img alt="Image" width="2000" height="1854" src="https://private-user-images.githubusercontent.com/121367647/530109836-b2f2a7fd-002d-4e28-9e81-09288bb850bf.png?jwt=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Njk1MzE0NjAsIm5iZiI6MTc2OTUzMTE2MCwicGF0aCI6Ii8xMjEzNjc2NDcvNTMwMTA5ODM2LWIyZjJhN2ZkLTAwMmQtNGUyOC05ZTgxLTA5Mjg4YmI4NTBiZi5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjYwMTI3JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI2MDEyN1QxNjI2MDBaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT0yNjFiNGNkNTdkYzk4ZGJiMzEzNzQ3ODIwM2FjNWJjYmRjODg3NzBlNTk0ZjcyZmM3ZjEyMGVjYzdjYWRhYmE0JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.FIjIMstxXQkfJadwDsL_18EAw7E-3S2mMFyP2I5yX3o">

I inspect and used contrast checker it really looks bad:

<img width="763" height="779" alt="Image" src="https://github.com/user-attachments/assets/a5c0394a-f932-4d3c-9ba0-c77df8a1179a" />

If apply same color from menu as foreground `#BEBFBF` it passes both WCAG level AA and AAA:

<img width="738" height="820" alt="Image" src="https://github.com/user-attachments/assets/175e18d8-1634-40e5-ab5b-7478c58498db" />

as burn doesn't have custom style it might be inject into the build, not 100% sure.
